sublime_jedi/tooltips/markdown.py

```diff
@@ -53,17 +53,20 @@ def _prepare_signature(self, docstring):
         """
         pattern = (
             '(?x)'
-            '^([\w\. \t]+\.[ \t]*)?'        # path
-            '(\w+)'                         # function / object
-            '[ \t]*(\([^\)]*\))'            # arguments
-            '(\s*->\s*.*$)?'                # annotation (rest of line)
+            '^([\w\. \t]+\.[ \t]*)?'    # path
+            '(\w+)'                     # function / object
+            '[ \t]*(\([^\)]*\))'        # arguments
+            '(?:'
+            '(\s*->\s*.*?)'             # annotation: -> Type
+            '(--|$)'                    # inline comment: -- blabla
+            ')?'
```

Comment:
This is what Jedi got me as a docstring for `L.extend()`. I was wondering why there was such strang highlighted text in the signature, when I found it to be a kind of inline comment beginning with `--`. It was already there before any string handling took place.